overlord/devicestate/firstboot.go

```diff
@@ -375,6 +384,34 @@ func importAssertionsFromSeed(st *state.State, sysLabel string, isCoreBoot bool)
 	return deviceSeed, nil
 }
 
+// processAutoImportAssertions attempts to load the auto import assertions 
+// and create all knows system users, if and only if the model grade is dangerous.
+// Processing of the auto-import assertion is opportunistic and should not fail
+// if model grade is dangerous, it attempts to load
+// auto import assertions and create all knows system users.
+// Processing of the auto-import assertion is opportunistic and should not fail
+func processAutoImportAssertions(st *state.State, deviceSeed seed.Seed, db asserts.RODatabase, commitTo func(batch *asserts.Batch) error) {
+	// if model is dangerous, check if there is auto-import.assert and import it
+	if deviceSeed.Model().Grade() != asserts.ModelDangerous {
+		return
+	}
+	seed20AssertionsLoader, ok := deviceSeed.(seed.AutoImportAssertionsLoaderSeed)
+	if !ok {
+		logger.Noticef("failed to auto-import assertions, invalid loader")
+		return
+	}
+	err := seed20AssertionsLoader.LoadAutoImportAssertions(commitTo)
+	if err != nil {
+		logger.Noticef("failed to auto-import assertions: %v", err)
+		return
+	}
+	serial, _ := findSerial(st, nil)
+	_, err = createKnownUsers(st, db, deviceSeed.Model(), serial, true)
```

Comment:
for clarity with booleans we tend to use constants, also probably we want a comment:

```
// automatic user creation is meant to imply sudoers
const sudoer = true
_, err = createKnownUsers(st, db, deviceSeed.Model(), serial, sudoer)
```

Reply:
Added